### PR TITLE
Shoaib answering comms edits2

### PR DESCRIPTION
**Pull request by shoaibsufi**
This changes merges in all the changes from the comms-edits2 branch in master

The tickets that have been closed with the comms-edits label are available here for reference - https://github.com/softwaresaved/event-organisation-guide/issues?q=is%3Aissue+is%3Aclosed+label%3A%22comms+edits%22+

Also - note some of the issues were discussed as part of a mini project when we had a tickets meeting - https://github.com/softwaresaved/event-organisation-guide/projects/1

Also the main template ticket is here - https://github.com/softwaresaved/event-organisation-guide/issues/72 - there is a link to the directory including the templates that needs to be checked processed outside of the pull request system.

### CODE REVIEW

File: docs/eog/eps-risk-management-assessment.rst [hunk 1]
```diff
@@ -2,21 +2,21 @@
 
 Risk Assessment
 ===============
-At the time of evaluation, the risks identified should be ranked by their likelihood and impact. The likelihood refers to the probability of risks actually happening, while impact is a score based on the most likely scenario and its ramifications after a potential risk happens. 
+At the time of evaluation, the risks identified should be ranked by their likelihood and impact. The likelihood refers to the probability of risks actually happening, while impact is the consequences if a potential risk happened.
```
Comment (JacalynLaird) [inline docs/eog/eps-risk-management-assessment.rst:5]: 'if a potential risk happens.'

File: docs/eog/eog-in-practice/cw20/infrastructure.rst [hunk 2]
```diff
@@ -31,12 +31,12 @@ We therefore obtained five Zoom Host accounts in total to accommodate this: one
...
 ^^^^^^^^^^^^
 
-Although Zoom has the chat functionality, it can be distracting for hosts/speakers and difficult to manage when there are many participants using it. 
+Although Zoom has chat functionality, it can be distracting for hosts/speakers and difficult to manage when there are many participants using it. 
 We decided to disable the Zoom chat (except for the case where participants could communicate directly with the meeting host) and direct engagement and communication to the CW20 Slack workspace, as this would allow participants to connect even when the Zoom room is closed. 
 A communication channel that persists outside the meeting is helpful for maintaining access to resources shared, referencing discussions and facilitating community engagement. 
 
 We created practical channels for event organisation and communication, such as a ``#help-desk`` channel for participants to direct their technical issues to, and private backchannels for the organisers, helpers, and Code of Conduct committee. 
-Participants were able to create channels for their discussion groups and Hackday teams. 
+Participants were able to create channels for their discussion groups and hack day teams.
```
Comment (selina-aragon) [inline docs/eog/eog-in-practice/cw20/infrastructure.rst:39]:
```suggestion
Participants were able to create channels for their discussion groups and Hack Day teams. 
```

Comment (selina-aragon) [reply docs/eog/eog-in-practice/cw20/infrastructure.rst:39]: In this instance, Hack Day refers to the one at CW.

File: docs/eog/eog-in-practice/cw20/infrastructure.rst [hunk 3]
```diff
@@ -75,4 +75,4 @@ Resources included that were used behind-the-scenes:
 Resources included that were participant-facing:
 
 - An example collaborative notes document (with agenda) to guide participants through the event, provide another pathway to engagement and compile questions, notes and outputs from the event.
-- The spreadsheets which we used to facilitate our unconference sessions (discussion groups, collaborative ideas and Hackday).
+- The spreadsheets which we used to facilitate our unconference sessions (discussion groups, collaborative ideas and hack day).
```
Comment (selina-aragon) [inline docs/eog/eog-in-practice/cw20/infrastructure.rst:78]:
```suggestion
- The spreadsheets which we used to facilitate our unconference sessions (discussion groups, collaborative ideas and Hack Day).
```

File: docs/eog/eog-in-practice/cw20/outputs-and-outcomes-plan.rst [hunk 4]
```diff
@@ -14,7 +14,7 @@ Outputs
 
 - `Speed blog posts <https://software.ac.uk/tags/cw20-speed-blog-posts>`_ from the discussion sessions
 - `Collaborative ideas <https://software.ac.uk/cw20/collaborative-ideas-group>`_
-- Hackday
+- Hack day
```
Comment (selina-aragon) [inline docs/eog/eog-in-practice/cw20/outputs-and-outcomes-plan.rst:17]:
```suggestion
- Hack Day
```

File: docs/eog/eog-in-practice/cw20/sponsorship.rst [hunk 5]
```diff
@@ -4,4 +4,4 @@ CW20 Sponsorship
 =================
 
 As soon as we made the decision to move online, we contacted the event's sponsors to notify them and offer to renegotiate any previously agreed packages. 
-We were extremely grateful that our sponsors `Microsoft <https://developer.microsoft.com/en-us/advocates/index.html>`_, `F1000 Research <https://f1000research.com/>`_, `Figshare <https://figshare.com/>`_, `eLife <https://elifesciences.org/>`_ and `Overleaf <https://www.overleaf.com>`_ pledged their full support of the event moving online and also engaged with the programme through lightning talks, mini-workshops and Hackday judging. 
+We were extremely grateful that our sponsors `Microsoft <https://developer.microsoft.com/en-us/advocates/index.html>`_, `F1000 Research <https://f1000research.com/>`_, `Figshare <https://figshare.com/>`_, `eLife <https://elifesciences.org/>`_ and `Overleaf <https://www.overleaf.com>`_ pledged their full support of the event moving online and also engaged with the programme through lightning talks, mini-workshops and hack day judging.
```
Comment (selina-aragon) [inline docs/eog/eog-in-practice/cw20/sponsorship.rst:7]:
```suggestion
We were extremely grateful that our sponsors `Microsoft <https://developer.microsoft.com/en-us/advocates/index.html>`_, `F1000 Research <https://f1000research.com/>`_, `Figshare <https://figshare.com/>`_, `eLife <https://elifesciences.org/>`_ and `Overleaf <https://www.overleaf.com>`_ pledged their full support of the event moving online and also engaged with the programme through lightning talks, mini-workshops and Hack Day judging. 
```

File: docs/eog/eog-in-practice/cw20/timeline.rst [hunk 6]
```diff
@@ -46,7 +46,7 @@ Communication
   - Links to the relevant Zoom room and note-taking document were sent via email through Eventbrite and Slack at the start of each day
    
 Resources
-  - We set up a uCONFLY instance for management of discussion group reports, mini-workship and demo session notes, collaborative ideas and Hackday ideas which also involved:
+  - We set up a uCONFLY instance for management of discussion group reports, mini-workship and demo session notes, collaborative ideas and hack day ideas which also involved:
```
Comment (selina-aragon) [inline docs/eog/eog-in-practice/cw20/timeline.rst:49]:
```suggestion
  - We set up a uCONFLY instance for management of discussion group reports, mini-workshop and demo session notes, collaborative ideas and Hack Day ideas which also involved:
```

File: docs/eog/eog-in-practice/cw20/timeline.rst [hunk 7]
```diff
@@ -154,4 +154,4 @@ Month 5 (August 2020)
 Month 6 (September 2020)
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Event Organisation Guide case study completed and reviewed
+- Event Organisation Guide In Practice indepth write up completed and reviewed
```
Comment (selina-aragon) [inline docs/eog/eog-in-practice/cw20/timeline.rst:157]:
```suggestion
- Event Organisation Guide In Practice in-depth write up completed and reviewed
```

File: docs/eog/eps-agenda.rst [hunk 8]
```diff
@@ -27,7 +27,7 @@ Exploring
 
 This includes sessions that help people work together to explore solutions or identify problems:
 
-* Collaborative ideas or hackday ideas sessions,
+* Collaborative ideas or hack day ideas sessions,
```
Comment (selina-aragon) [inline docs/eog/eps-agenda.rst:30]:
```suggestion
* Collaborative ideas or Hack Day ideas sessions,
```

File: docs/eog/eps-risk-management-assessment.rst [hunk 9]
```diff
@@ -2,22 +2,22 @@
...
+Once the risks are assessed based on their likelihood and impact score, they can be visualised on a risk matrix.
 
 The following is an example risk matrix from the `Wikimedia Commons <https://commons.wikimedia.org/wiki/File:IC-Risk-Assessment-Matrix-Template.jpg>`_. The image is available under the CC-BY-SA 4.0 license).
 
 .. figure:: img/536px-IC-Risk-Assessment-Matrix-Template.jpg
   :alt: Risk assessment matrix
 
-The event team can then decide which part of the matrix is above or below the project risk threshold. The risk threshold can be set according to the experience of the team or the degree of change in the type of event compared to others. New, complex and larger scale events will probably call for a lower risk threshold. The risks above the threshold will be considered as mitigated risks. 
+The event team can then decide which part of the matrix is above or below the project risk threshold. For experienced teams the risk threshold can be set high. The threshold may need to be lowered if the team is new or lacks experience in running a particular type of event. Mitigation plans will need to be formulated for risks above the threshold.
```
Comment (selina-aragon) [inline docs/eog/eps-risk-management-assessment.rst:19]:
```suggestion
The event team can then decide which part of the matrix is above or below the project risk threshold. For experienced teams, the risk threshold can be set high. The threshold may need to be lowered if the team is new or lacks experience in running a particular type of event. Mitigation plans will need to be formulated for risks above the threshold. 
```

**Review by selina-aragon — CHANGES_REQUESTED** [review]
I'm happy with the overall changes made to the EOG. I've added very minor edits. Once those are done, we're ready for the guide to be publicised.

File: docs/eog/eps-risk-management-assessment.rst (hunk 1 above)
Comment (shoaibsufi) [reply docs/eog/eps-risk-management-assessment.rst]: made the update - see commit below - https://github.com/softwaresaved/event-organisation-guide/pull/75/commits/9f4eb537e07f18159ff4951a2b5ccc402cd0744b

**Comment by shoaibsufi** [comment]
It’s ready for review @Selina - I have accepted all suggested changes and made a change via commit as suggested by @Jacalyn Laird also for ticket #72 I actioned the changes @Jacalyn Laird suggested and the files are now uploaded as new versions to Zenodo - the url's used in EOG resolve to the latest version so we are good there.

**Review by selina-aragon — APPROVED** [review]
All good now. We are ready to publicise 👍

**Comment by shoaibsufi** [comment]
@selina-aragon has approved so merging with master

**Pull request merged**